edx_event_bus_kafka/internal/producer.py

```diff
@@ -284,13 +284,17 @@ def send(
         context = ProducingContext(signal=signal, initial_topic=topic, event_key_field=event_key_field,
                                    event_data=event_data, event_metadata=event_metadata)
         try:
-            context.event_data_as_json = json.dumps(get_signal_serializer(signal).to_dict(event_data))
-            context.event_metadata_as_json = event_metadata.to_json()
+            # Please keep these context assignments in order from least-risky to most-risky so that we
```

Comment:
I don't understand why the order matters, or what we mean by "risky"

Reply:
Can we change "risky" to "likely to error"? I think that would be clearer

Reply:
Definitely can change that!